Documentation~/ref_actions.md

```diff
@@ -1,17 +1,74 @@
 # Action reference
 
-This table lists all the actions available in ProBuilder, and indicates which [Edit modes](modes.md) support them.
+This table lists all the actions you can apply to a ProBuilder object, and indicates which [Edit modes](modes.md) support them. 
+
+These actions are available:
+
+* From the **main menu** > **Tools** > **ProBuilder**. The actions are grouped in **Selection**, **Interaction**, **Object**, and **Geometry**.
+* From the context menu: 
+    1. In the **Tools** overlay, set the working context to ProBuilder. 
+    1. In the **Tool Settings** overlay, select an edit mode.
+    1. In the **Scene** view, right-click on a ProBuilder object.
+
+<!--So... I divided this page by category. I think it's easier to scan. But then the question is: should the TOC match, or should the TOC stay alphabatised? -->
```

Comment:
Good question. I think we should make the TOC match. If a user is going through it alphabetically, then the implication is that they know what they're looking for. If they know what they're looking for, why wouldn't they use CTRL+F?

Reply:
Oh man was that old news :)